app/templates/admin/userManagement.html

```diff
@@ -166,7 +167,40 @@ <h5>Programs</h5>
       </div>
     </div>
   </div>
-  
+{% endif %}
+{% if g.current_user.isProgramManager() and not g.current_user.isCeltsAdmin %}
+<div class="container-fluid">
+  <div class="row justify-content-center">
+    <div class="col-lg-8 col-md-10 col-sm-12">
+      <h1 class="text-center mb-5">Program Management</h1>
+      
+      <div class="d-flex flex-column align-items-center">
+
+        {% for program in programs %}
+        <div class="d-flex justify-content-between align-items-center mb-3 w-100 p-3 border rounded">
+          <div class="flex-grow-1 text-start">
+            <span class="form-label borderless fs-5">{{ program.programName }}</span>
+          </div>
+          <div class="d-flex gap-2">
+            <button class="btn btn-success btn-sm editDetails" 
+                    role="button" 
+                    id="programManagementButton{{ loop.index }}"  
+                    loop-index="{{ loop.index }}"
+                    data-programid="{{program.id}}">
+                <span style="display: none;" id="editDetailsButtonSpinner{{ loop.index }}">
+                  <span class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
+                  <span class="sr-only">Loading...</span>  
```

Comment:
No need for Loading and spinning. It seems inconsistent with the functionality in Program Management for the admin

Reply:
I don't see any changes. Did you commit and push?